app/frontend/src/components/designer/FormDisclaimer.vue

```diff
@@ -1,17 +1,23 @@
 <template>
   <div>
-    <h3 class="mb-5">
+    <h3 class="mb-5" :class="{ 'dir-rtl': isRTL }">
       {{ $t('trans.formDisclaimer.disclaimerAndStatement') }}
     </h3>
-    <ul>
+    <ul :class="isRTL ? 'mr-6' : null">
       <li>
-        {{ $t('trans.formDisclaimer.privacyLaw') }}
+        <span :class="{ 'dir-rtl': isRTL }">
+          {{ $t('trans.formDisclaimer.privacyLaw') }}
+        </span>
       </li>
       <li>
-        {{ $t('trans.formDisclaimer.disclosure') }}
+        <span :class="{ 'dir-rtl': isRTL }">
+          {{ $t('trans.formDisclaimer.disclosure') }}
+        </span>
       </li>
       <li>
-        {{ $t('trans.formDisclaimer.consent') }}
+        <span :class="{ 'dir-rtl': isRTL }">
+          {{ $t('trans.formDisclaimer.consent') }}
+        </span>
       </li>
       <li>
         {{ $t('trans.formDisclaimer.formIntention') }}
```

Comment:
I'm confused here - the formIntention doesn't have the RTL but it seems to be working? Maybe you don't need it on every <li> because the <ul> does it for all of the <li> items?

Reply:
Oh, nevermind.... the `ul` doesn't set `dir-rtl`. So why is this working for `formIntention`?

Reply:
Is it working because the Create.vue page that is using the FormDisclaimer already has RTL set?

My concern is that we have these `dir-rtl` classes in so many places, but maybe they aren't needed? Maybe it's just on the outer containers that we need it set?

Reply:
We are using Vuetify components, and I have to add dir-rtl to each Vuetify component so it works correctly.

Reply:
But for this page being just HTML - if you put `:class="{ 'dir-rtl': isRTL }"` _only_ on the `div` on line 2, will it still work correctly for all the tags under it? (just curious!)

Reply:
Actually, no - if you remove _all_ the `:class="{ 'dir-rtl': isRTL }"` from this file, will it still work because it's included in another page? (again just curious - I'm hoping we can make things as simple as possible, so that there is less code to maintain in the future).

Reply:
I have fixed that. I am also scanning through other components too.

Reply:
There are some pages/components that I only apply dir-RTL to the outer container while some I avoided the outer container because it will affect the layout for example in FormDesigner.vue